vhdbuilder/packer/install-dependencies.sh

```diff
@@ -153,6 +153,9 @@ if [[ ${CONTAINER_RUNTIME:-""} == "containerd" ]]; then
     downloadCrictl ${CRICTL_VERSION}
     echo "  - crictl version ${CRICTL_VERSION}" >> ${VHD_LOGS_FILEPATH}
   done
+  
+  KUBERNETES_VERSION=$(echo -e "$CRICTL_VERSIONS" | tail -n 2 | head -n 1 | tr -d ' ') installCrictl || exit $ERR_CRICTL_DOWNLOAD_TIMEOUT
```

Comment:
As this uses the latest version of crictl, I was curious about the crictl version skew/compat with containerd and k8s versions and found this: https://github.com/kubernetes-sigs/cri-tools/issues/297. Seems like it should be fine.